R/scripts/full_filter.R

```diff
@@ -1,11 +1,13 @@
+suppressMessages(library(parallel))
+
 option_list = list(
-  optparse::make_option(c("-c", "--config"), action="store", default=Sys.getenv("CONFIG_PATH"), type='character', help="path to the config file"),
-  optparse::make_option(c("-p", "--pipepath"), action="store", type='character', help="path to the COVIDScenarioPipeline directory", default = "./"),
-  optparse::make_option(c("-s", "--scenarios"), action="store", default='all', type='character', help="name of the intervention to run, or 'all' to run all of them"),
-  optparse::make_option(c("-d", "--deathrate"), action="store", default='all', type='character', help="name of the death scenarios to run, or 'all' to run all of them"),
-  optparse::make_option(c("-j", "--jobs"), action="store", default="72", type='integer', help="Number of jobs to run in parallel"),
-  optparse::make_option(c("-k", "--sims_per_slot"), action="store", default=NA, type='integer', help = "Number of simulations to run per slot"),
-  optparse::make_option(c("-n", "--slots"), action="store", default=NA, type='integer', help = "Number of slots to run."),
+  optparse::make_option(c("-c", "--config"), action="store", default=Sys.getenv("COVID_CONFIG_PATH", Sys.getenv("CONFIG_PATH")), type='character', help="path to the config file"),
+  optparse::make_option(c("-p", "--pipepath"), action="store", type='character', help="path to the COVIDScenarioPipeline directory", default = Sys.getenv("COVID_PATH", "COVIDScenarioPipeline/")),
```

Comment:
Should this be COVID_PATH?

Reply:
not sure I understand, are you suggesting an alternative name vs COVID_PATH?